README.md

```diff
@@ -205,6 +212,7 @@ An action to generate SLSA build provenance for an artifact
 | tag_name | The github release to generate provenance on.
 (if set the artifacts will be downloaded from the release and the provenance will be added as an additional release asset.)
  | `false` |  |
+| extra_materials | paths to files that contain in-toto materials to include in the provenance | `false` | |
```

Comment:
nit:
1. This help string is in three different places, but it's different in all three. It would be nice to use the same string consistently.
2. It would help to say that these are files are JSON arrays in the same format as provenance v0.1 `materials`, or something like that. Otherwise it's not clear what exactly the format is.